c/json.c

```diff
@@ -372,14 +392,7 @@ void jsonConvertAndWriteBuffer(jsonPrinter *p, char *text, size_t len,
       DUMPBUF(text, len);
     }
     if (escape) {
-      size_t bytesWritten;
-
-      bytesWritten = writeBufferWithEscaping(p, len, text);
-      if (bytesWritten < 0) {
-        ERROR("jsonConvertAndWriteBuffer() error: bytesWritten = %d\n",
-                bytesWritten);
-        return;
-      }
+      writeBufferWithEscaping(p, len, text);
```

Comment:
Should we keep the check? Looks like the call can fail. The only fix would be changing `size_t` to `ssize_t`.